DifferenceGenerator/src/test/kotlin/DifferenceGeneratorTest.kt

```diff
@@ -182,4 +184,51 @@ internal class DifferenceGeneratorTest {
             DifferenceGenerator(video9Frames, compressedVideo, outputPath, algorithm)
         }
     }
+
+    @Test
+    fun `test masking`() {
+        val outputPath = resourcesPathPrefix + "outputDeletion.mov"
```

Comment:
maybe you could use a unique file name for the mask test here so all tests have separate output videos